src/components/Link.react.js

```diff
@@ -2,69 +2,72 @@
 
 import PropTypes from 'prop-types';
 
-import React, {Component} from 'react';
+import React, { Component } from 'react';
```

Comment:
Similarly here - this isn't enforced in eslint (although it should be), but our convention has been no spaces around `{}` imports